modules/distribution/src/repository/resources/conf/default.json

```diff
@@ -363,7 +363,428 @@
   "identity.events.schemes.OIDCLogoutHandler.properties.enable": false,
   "identity.events.schemes.confirmationCodesInvalidate.subscriptions": [
     "POST_DELETE_USER"
-  ]
+  ],
+
+
+
+  "identity.clock_skew": "300",
+  "jdbc.persistence_manager.skip_db_schema_creation": false,
+  "session_data.persist.enable": true,
+  "session_data.persist.temp_data": true,
+  "session_data.persist.db_pool_size": "0",
+  "session_data.cleanup.enable_for": ["expired_data","invalidated_data","temp_data"],
+  "session_data.cleanup.expired_data.expiry_time_mins": "20160",
+  "session_data.cleanup.expired_data.frequency_mins": "1440",
+  "session_data.cleanup.expired_data.in_chunks_of": "50000",
+  "session_data.cleanup.temp_data.db_pool_size": "20",
+  "session_data.cleanup.temp_data.expiry_time_mins": "40",
+
+  "session.timeout.idle_session_timeout_mins": "15",
+  "session.timeout.remember_me_session_timeout_mins": "20160",
+
+  "key_mgt.keystore_dir": "${carbon.home}/conf/keystores",
+  "key_mgt.key_manager_type": "$ref{keystore.tls.type}",
+  "key_mgt.trust_manager_type": "$ref{truststore.type}",
+
+  "identity.issuer_policy": "SelfAndManaged",
+  "identity.token_validation_policy": "CertValidate",
+
+  "oauth.token.cleanup.enable": true,
+  "oauth.token.cleanup.backup_for_auditing": true,
+  "oauth.token.persist_access_token_alias": true,
+  "oauth.use_entityid_as_issuer_in_oidc_discovery": true,
+  "oauth.authz_code.validity_time_s": "300",
+  "oauth.app_token.validity_time_s": "3600",
+  "oauth.user_token.validity_time_s": "3600",
+  "oauth.refresh_token.validity_time_s": "84600",
```

Comment:
let's get rid of '_s' suffix from key, and have the unit at value.

Refer : https://github.com/wso2/carbon-kernel/pull/1982

Reply:
Fixed.